test/specs/multipleBrowsersDocumentUpload.js

```diff
@@ -5,15 +5,15 @@ const options = {
   pageObjects: ['Confirm', 'DocumentSelector', 'Welcome', 'DocumentUpload']
 }
 
-describe('DOCUMENT UPLOAD ON IE11', options, ({driver, pageObjects}) => {
+describe('DOCUMENT AND PDF UPLOAD ON MULTIPLE BROWSERS', options, ({driver, pageObjects}) => {
   const {welcome, documentSelector, confirm, documentUpload} = pageObjects
 
-  it('should upload document on IE11 browser', async () => {
+  it('should upload document with jgp', async () => {
     goToPassportUploadScreen(driver, welcome, documentSelector)
     uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
   })
 
-  it('should upload PDF on IE11 browser', async () => {
+  it('should upload document with pdf', async () => {
```

Comment:
Minor: should be all caps since it's an abbreviation

Reply:
Done